### DISCUSSION TITLE
Allow Future Scheduling of Voucher Emails

### DISCUSSION THREAD

**Discussion by theschweitz** [pretix/pretix#4245] [Ideas]
Currently you can schedule emails to go out to specific product types, check-in status, etc. 
And you can send vouchers via email in the Multiple Vouchers flow.
Users should be able to create the email that will go out with multiple vouchers, and schedule that to send, in the same way other emails are scheduled to send!

**Comment by jxs714** [comment]
> Currently you can schedule emails to go out to specific product types, check-in status, etc. And you can send vouchers via email in the Multiple Vouchers flow. Users should be able to create the email that will go out with multiple vouchers, and schedule that to send, in the same way other emails are scheduled to send!

I agree. There should also be an option for you to send vouchers to any email outside of your orders for guests that you provide comp tickets too. Otherwise you have to contact them outside of the software to provide it
